tests/components/test_graphite.py

```diff
@@ -19,21 +19,18 @@ class TestGraphite(unittest.TestCase):
     def setup_method(self, method):
         """Setup things to be run when tests are started."""
         self.hass = get_test_home_assistant()
-        self.hass.config.latitude = 32.87336
-        self.hass.config.longitude = 117.22743
         self.gf = graphite.GraphiteFeeder(self.hass, 'foo', 123, 'ha')
 
     def teardown_method(self, method):
         """Stop everything that was started."""
         self.hass.stop()
 
-    @mock.patch('homeassistant.components.graphite.GraphiteFeeder')
-    def test_minimal_config(self, mock_gf):
-        """Test setup with minimal configuration."""
-        self.assertTrue(graphite.setup(self.hass, {}))
-        mock_gf.assert_called_once_with(self.hass, 'localhost', 2003, 'ha')
+    @patch('homeassistant.components.graphite.GraphiteFeeder')
+    def test_setup(self, mock_gf):
+        """Test setup."""
+        assert not bootstrap.setup_component(self.hass, 'graphite', {})
```

Comment:
If you assert that `bootstrap.setup_component` returns False, it means that you are assuming the setup is failing. However the test name makes it seem like you want to test setup.

The reason why validation si failing because you pass in an empty dict as config instead of wrapping it in a dict like as if it was in the config file:

```python
{
  'graphite': {}
}
```